src/Controller/DefaultController.php

```diff
@@ -14,12 +14,20 @@
 use App\Entity\PersonResearchGroup;
 use App\Util\FundingOrgFilter;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 
 /**
  * This is the default controller.
  */
 class DefaultController extends AbstractController
 {
+    private $params;
+
+    public function __construct(ParameterBagInterface $params)
+    {
+        $this->params = $params;
+    }
+
```

Comment:
The `ParamaterBagInterface` already exists in the `AbstractController` (actually `ContainerBagInterface`). So no need to add it again.

Just use `$this->getParameter('SITEMAP_MIN_DATE')`

Also $params is not used anywhere.